google-cloud-spanner/acceptance/spanner/client/read_test.rb

```diff
@@ -211,4 +211,40 @@
                                                                               { id: 11 },
                                                                               { id: 12 }]
   end
+
+  it "accepts the order_by option" do
+    _(db.read(table_name,
+              [:id],
+              order_by: Google::Cloud::Spanner::V1::ReadRequest::OrderBy::ORDER_BY_NO_ORDER)
+              .rows.map(&:to_h)).must_equal [{ id: 1 },
```

Comment:
Nit: can you indent this further so it's even with the start of the `Google::Cloud::Spanner...` in the previous line, so that it's more clear this is all part of the same expression being assigned to `order_by`